desktop/desktop.scss

```diff
@@ -77,6 +77,11 @@
     }
 }
 
+//remove stray border from bottom of last post
+.topic-status-info {
+    border-top: 0;
```

Comment:
Looks like there is alot of copy-pasta code in there. 4 spaces are fine for now. I'll clean up the code soon. :+1: